src/main/java/com/google/devtools/build/lib/bazel/rules/python/BazelPyBinaryRule.java

```diff
@@ -99,7 +99,7 @@ public Metadata getMetadata() {
 py_binary(
     name = "test_main",
     srcs = ["test_main.py"],
-    deps = [":testlib"],
+    deps = [":testing"],
```

Comment:
The example seems to be showing a java_library wanting to use a py_binary. If you look below, the java_library has this py_binary in data.

I dunno what this deps line is for. There doesn't appear to be any definition of the "testlib" target. So lets just remove it entirely.

```suggestion
```